desktop-dde/dtk6declarative/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=dtk6declarative
+PKGSEC=dde
+PKGDEP="dtk6core dtk6gui gcc-runtime glibc librsvg qt-6"
+BUILDDEP="cmake doxygen gtest"
+PKGDES=" a widget development toolkit based on QtQuick/QtQml"
```

Comment:
Widget development toolkit for deepin based on QtQuick

QtQML is redundant in this context.